configure.ac

```diff
@@ -59,6 +59,24 @@ else
 	AC_DEFINE([HAVE_DWARF],[1],[DWARF debug_frame support in libunwind])
 fi
 
+echo -n "Checking for ELF_C_READ_MMAP... "
```

Comment:
Why not use `AC_CHECK_DECLS()` ?

Like
```
AC_CHECK_DECLS([ELF_C_READ_MMAP1], [], [], [#include <libelf.h>])
```

Reply:
Yes, it looks much better